modules/crm_core_default_matching_engine/config/install/crm_core_match.matcher.household.yml

```diff
@@ -0,0 +1,9 @@
+langcode: en
```

Comment:
Is it really useful to still have that default module? Maybe we should just merge everything into the match module?

I don't crm_core very well. Those three types are just default config too I guess, there could be others. Do they really still make sense to have, especially without actual configuration?